src/dsp_tools/xmllib/models/dsp_base_resources.py

```diff
@@ -62,9 +62,9 @@ def create_new(
             comments: the comment(s) that this annotation consists of, i.e. the annotation itself (cardinality 1-n)
             permissions: permissions of this annotation resource
 
-        Warnings:
-            - If the Resource ID is not a valid string
-            - If the label is not a valid string
+        Warns:
+            Warning: If the Resource ID is not a valid string
+            Warning: If the label is not a valid string
```

Comment:
This is necessary in order to be parsed and rendered correctly. The previous syntax triggered a warning in the docstrings parser, which made the CI fail.

Reply:
I just chose the very generic superclass `Warning`, which is always true.